.github/workflows/build.yml

```diff
@@ -40,13 +40,13 @@ jobs:
             build-net-tokio: true
             build-no-std: true
             build-futures: true
-            build-tx-sync: true
+            build-tx-sync: false
```

Comment:
Now disabled building on windows. While building itself should be fine, testing doesn't work as the `ElecrtrsD`/`BitcoinD` crates don't support Windows. Let me now if I should split building/testing in two different variables to enable more finer grained control. So far, I'm thinking checking that it builds on Windows is not worth the additional noise in the CI script.